openrouteservice-api-tests/src/test/java/org/heigit/ors/v2/services/routing/ResultTest.java

```diff
@@ -1944,8 +1944,8 @@ public void testWheelchairInclineRestriction() {
                 .then()
```

Comment:
they a re basically testing if the respective API option such as restrictions have any effect, sending the same request with and without that option to make sure the result changes. Might be best to check these numbers against each other, on the other hand this also lets us notice if route results change unexpectedly...

Reply:
Ok. Can we make this a bit clearer in the code?

Reply:
Any suggestions how?

Reply:
O.K., I think, this would be too much for this PR. Probably other parts of the test-suite are affected by the same problems and we have to review all tests.